default_settings.json

```diff
@@ -0,0 +1,5254 @@
+	//  - newWindow: Always open untrusted files in a separate window in restricted mode without prompting.
+	"security.workspace.trust.untrustedFiles": "prompt",
+
+	// Controls the behavior of clicking an activity bar icon in the workbench.
+	//  - toggle: Hide the side bar if the clicked item is already visible.
+	//  - focus: Focus side bar if the clicked item is already visible.
+	"workbench.activityBar.iconClickBehavior": "toggle",
+
+	// Controls the visibility of the activity bar in the workbench.
+	"workbench.activityBar.visible": true,
+
+	// Overrides colors from the currently selected color theme.
+	"workbench.colorCustomizations": {},
+
+	// Specifies the color theme used in the workbench.
+	"workbench.colorTheme": "Gitpod Light",
+
+	// Controls the number of recently used commands to keep in history for the command palette. Set to 0 to disable command history.
+	"workbench.commandPalette.history": 50,
+
+	// Controls whether the last typed input to the command palette should be restored when opening it the next time.
+	"workbench.commandPalette.preserveInput": false,
+
+	// If an editor matching one of the listed types is opened as the first in an editor group and more than one group is open, the group is automatically locked. Locked groups will only be used for opening editors when explicitly chosen by user gesture (e.g. drag and drop), but not by default. Consequently the active editor in a locked group is less likely to be replaced accidentally with a different editor.
+	"workbench.editor.autoLockGroups": {
+		"default": false,
+		"workbench.input.interactive": false,
+		"interactive": false,
+		"workbench.editorinputs.searchEditorInput": false,
+		"terminalEditor": true,
+		"jupyter-notebook": false,
+		"imagePreview.previewEditor": false,
+		"vscode.markdown.preview.editor": false,
+		"jsProfileVisualizer.cpuprofile.table": false,
+		"jsProfileVisualizer.heapprofile.table": false,
+		"mainThreadWebview-markdown.preview": false
+	},
+
+	// Controls if the centered layout should automatically resize to maximum width when more than one group is open. Once only one group is open it will resize back to the original centered width.
+	"workbench.editor.centeredLayoutAutoResize": true,
+
+	// Controls the behavior of empty editor groups when the last tab in the group is closed. When enabled, empty groups will automatically close. When disabled, empty groups will remain part of the grid.
+	"workbench.editor.closeEmptyGroups": true,
+
+	// Controls whether editors showing a file that was opened during the session should close automatically when getting deleted or renamed by some other process. Disabling this will keep the editor open  on such an event. Note that deleting from within the application will always close the editor and that editors with unsaved changes will never close to preserve your data.
+	"workbench.editor.closeOnFileDelete": false,
```

Comment:
I think this should be `true`.

Reply:
Interesting. I expect it to be pretty rare that students delete files but `true` is probably is the expected behavior.